lstchain/scripts/lstchain_data_r0_to_dl1.py

```diff
@@ -47,6 +47,12 @@
                     default=None
                     )
 
+parser.add_argument('--max_events', '-maxevts', action='store', type=int,
+                    dest='max_events',
+                    help='Maximum number of events to be processed.',
+                    default=int(1e15)
+                    )
```

Comment:
`max_events`  is already an entry in the config file, is it really necessary to add another argument to the script?

Reply:
Not absolutely necessary, but I find it easier for tests to have it also as a command-line option. I guess it does not hurt...